package.json

```diff
@@ -7,7 +7,7 @@
     "start-message": "babel-node tools/startMessage.js",
     "prestart": "npm-run-all --parallel start-message remove-dist",
     "start": "npm-run-all --parallel start-api test:watch open:src lint:watch",
-    "start-api": "sh script/test-server",
+    "start-api": "script/test-server",
```

Comment:
I added the `sh` in [`bbceb54`](https://github.com/jekyll/jekyll-admin/commit/bbceb548ec5fffa795088ad97956d90056504aae) as I required that addition on my Windows system